packages/stores/src/account/base.ts

```diff
@@ -759,6 +785,8 @@ export class AccountStore<Injects extends Record<string, any>[] = []> {
           signDoc
         ));
 
+    console.log("signed", signed);
```

Comment:
Note: I think that the problem stems from the outdated call to dependencies on the few lines above